frontend/containers/SurplusContainer.vue

```diff
@@ -78,6 +95,12 @@ export default Vue.extend({
                 }
             },
         },
+        selectedAuction(): SurplusAuction | undefined {
+            return this.auctions.find((auction: SurplusAuction) => auction.id === this.selectedAuctionId);
+        },
```

Comment:
isn't auctions a dict? it's if it's mapped from getters, then i assume it's the same as in store which is `auctionStorage: Record<string, AuctionTransaction>;`